azure-pipelines.yml

```
@@ -32,7 +32,18 @@ jobs:
     - script: sudo apt-get install -qy --force-yes graphviz
       displayName: 'Install GraphViz'
 
-    - script: make html
+    - script: |
+        wget -O drawio.deb https://github.com/jgraph/drawio-desktop/releases/download/v12.6.5/draw.io-amd64-12.6.5.deb
+        sudo apt-get install libasound2 ./drawio.deb
```

```suggestion
        sudo apt-get install -qy --force-yes libasound2 ./drawio.deb
```
this is what the other commands do

---

yes, that was the plan I just didn't get to that cuz I prioritized getting stuff working first